apps/native/app/src/index.tsx

```diff
@@ -53,10 +55,11 @@ async function startApp() {
     // Set the app root
     await Navigation.setRoot({ root })
 
+    handleInitialUrl()
```

Comment:
After rendering the one tab at a time the initial deepling url logic for wallet was broken. This fixes that